gapic-generator-cloud/templates/cloud/client/_client.erb

```diff
@@ -81,32 +98,20 @@ class <%= client_name service %>
 
   def default_settings client_config, timeout, metadata, lib_name,
                         lib_version
-    package_version = Gem.loaded_specs["<%= gem_name api %>"]
-      .version.version
+    package_gem = Gem.loaded_specs["<%= gem_name api %>"]
+    package_version = package_gem ? package_gem.version.version : nil
 
     google_api_client = ["gl-ruby/#{RUBY_VERSION}"]
```

Comment:
Bah I originally wrote this and it looks ugly. Can you make this look nicer by using `.join " "`

Reply:
Done.